src/Locks.ts

```diff
@@ -126,13 +120,20 @@ export default class Locks extends EventEmitter<LockEventMap> {
 
   async release(id: string): Promise<void> {
     const self = await this.space.members.getSelf();
+
     if (!self) {
       throw ERR_NOT_ENTERED_SPACE();
     }
 
-    this.deleteLock(id, self.connectionId);
+    const lock = this.getLock(id, self.connectionId);
+    if (!lock) return;
 
-    await this.updatePresence(self);
+    lock.status = 'unlocked';
+    lock.reason = undefined;
+    // Send presence update with the updated lock, but delete afterwards so when the
+    // message is processed an update event is fired (the state of the )
```

Comment:
Looks like part in parenthesis hasn't completed ( and looks like without it it's clear enough :) )